### PR TITLE
Release 2.2.1

### PR DESCRIPTION
**Pull request by kit-ty-kate**
~This is a draft until it is decided whether or not https://github.com/ocaml/opam/pull/6169 should be part of the release or whether it should be part of 2.2.2~

### CODE REVIEW

**Review by rjbou — APPROVED** [review]
Thanks!

File: CHANGES [hunk 1]
```diff
@@ -3,6 +3,24 @@ repositories (changes that are automatically handled by the format upgrade tools
 are not marked). Those prefixed with "(+)" are new command/option (since
 2.1.0~alpha2).
 
+2.2.1:
+* (*) Fix a regression in `opam install --deps-only` where the direct
+  dependencies were not set as root packages [#6125 @rjbou]
+* (*) Fix a regression when building git packages where the build result
+  were sometimes inconsistent compared to outside opam, by disabling shallow
+  clone by default except when fetching an opam repositories
```
Comment (rjbou) [inline CHANGES:11]:
```suggestion
* (*) Fix a regression on git fetching by disabling shallow
  clone by default except when fetching an opam repositories
```
It is not only git package build that was affected

Comment (kit-ty-kate) [reply CHANGES:11]: Do you have an example that isn't a git package?

Comment (kit-ty-kate) [reply CHANGES:11]: oh you mean local (rsync), sure. I can replace by "packages using git"

Comment (kit-ty-kate) [reply CHANGES:11]: mh, actually no the issue wouldn't be triggered there. Do you have an example?

Comment (kit-ty-kate) [reply CHANGES:11]: The entry doesn't say it's failing. Only that the result is inconsistent between with and without opam. That does englobe all that in my opinion

Comment (kit-ty-kate) [reply CHANGES]: fixed

**Pull request merged**